spec/wings/hydra/works/services/add_file_to_file_set_spec.rb

```diff
@@ -34,8 +34,7 @@
         expect(ids.first.to_s).to start_with "#{file_set.id}/files/"
 
         file_metadata = Hyrax.custom_queries.find_file_metadata_by(id: ids.first)
-        expect(file_metadata.content.first).to start_with('%PDF-1.3')
-        expect(file_metadata.mime_type.first).to eq pdf_mimetype
+        expect(file_metadata.mime_type).to eq pdf_mimetype
```

Comment:
It seems good to remove content from the file_metadata.  I'm wondering if the content is tested somewhere else.  I ask because there were lots of issues to work through related to content and the need to translate it back and forth between the Valkyrie FileMetadata object and the AF metadata_node.  It could be there are tests under the AF converter tests and I just haven't seen them yet.